app/components/champ-card.tsx

```diff
@@ -0,0 +1,20 @@
+import { Center, Flex, Heading } from "@chakra-ui/react";
+import ChampIcon from "./champ-icon";
+import champions from "../_data/champions";
+
+function ChampCard({ championId }: { championId: string }) {
+  return (
+    <Flex>
+      <Center pe={2}>
+        <ChampIcon champion={championId} />
+      </Center>
+      <Center>
+        <Heading size="md">
+          {championId ? champions.data[championId].name : '-'}
+        </Heading>
+      </Center>
+    </Flex>
+  );
+}
```

Comment:
To reduce the logic of adding blank data to each child element, better to have one blank item conditional

```suggestion
function ChampCard({ championId }: { championId: string }) {
 if(!championId) {
  return (insert blank champ card)
 }

  return (
    <Flex>
      <Center pe={2}>
        <ChampIcon champion={championId} />
      </Center>
      <Center>
        <Heading size="md">
          {championId ? champions.data[championId].name : '-'}
        </Heading>
      </Center>
    </Flex>
  );
}
```